src/compas_ags/ags/graphstatics.py

```diff
@@ -196,6 +199,56 @@ def form_count_dof(form):
     return int(k), int(m)
 
 
+def form_compute_nullspace(form, force, constraints=None):
+    r"""Compute the null space of a form diagram assuming a set of constraints.
+    It returns the new position of the form diagram that represents how the
```

Comment:
when we use the `compas.numerical.nullspace` it returns unit vectors with the "null" solution of the system Ax = 0. Therefore this is the "scale" of the displacement field, it will always be a unit vector...